jax/_src/ad_checkpoint.py

```diff
@@ -203,13 +205,46 @@ def checkpoint(fun: Callable, prevent_cse: bool = True,
   ...     f2 = recursive_checkpoint(funs[len(funs)//2:])
   ...     return lambda x: f1(jax.checkpoint(f2)(x))
   ...
+
+  If ``fun`` involves Python control flow which depends on argument values,
+  it may be necessary to use the ``static_argums`` parameter. For example,
```

Comment:
```suggestion
  If ``fun`` involves Python control flow that depends on argument values,
  it may be necessary to use the ``static_argnums`` parameter. For example,
```